src/main/java/SQLiteSource.java

```diff
@@ -580,6 +582,116 @@ public Accounts login(String username, String password) throws DataSourceExcepti
         }
     }
 
+    @Override
```

Comment:
Add state requirement > Function only adds developer or user in one call. throws Exception if dev & user are both not null.
--OR--
Update so that it tries both & returns some identifier of state for each account type